src/utils/value_parsers.rs

```diff
@@ -0,0 +1,9 @@
+use std::error::Error;
+
+/// Parse key:value pair from string, used as a value_parser for Clap arguments
+pub fn kv_parser(s: &str) -> Result<(String, String), Box<dyn Error + Send + Sync>> {
```

Comment:
Following offline discussion, this should return a `Result<(String, String), CustomErrorType>`